test/muleCI/mule.yaml

```diff
@@ -237,11 +238,11 @@ jobs:
       - stash.Unstash.packages
       - docker.Make.archive.amd64
       - docker.Make.archive.amd64.centos8
```

Comment:
archive actually uploads everything, so we only need one of these tasks

Reply:
OK, very good, removed.